bin/cli.js

```diff
@@ -367,8 +351,7 @@ cli.on('requireFail', function(name) {
 cli.launch(
   {
     cwd: argv.cwd,
-    knexfile: argv.knexfile,
-    knexpath: argv.knexpath,
+    configPath: argv.knexfile,
```

Comment:
The `Liftoff#launch(..)` method does not understand the `knexfile` and `knexpath` parameters.  Furthermore, it does not pass them though to the `env` object when calling the `invoke(..)` callback.